hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestSecondaryIndexPruning.scala

```diff
@@ -516,8 +515,6 @@ class TestSecondaryIndexPruning extends SparkClientFunctionalTestHarness {
            |FROM hudi_metadata('$basePath')
            |WHERE type=7
        """.stripMargin)(
-        Seq("abc", "row1", true),
-        Seq("cde", "row2", true),
```

Comment:
Why removing this?  Are the expected results changed?

Reply:
yes. since we fixed the how cleaning impacts MDT, where we delete the record instead of just holding onto the key, and null stat value. hence had to fix the expected values in these tests.